eng/config/PublishData.json

```diff
@@ -210,7 +210,8 @@
       "channels": [],
       "vsBranch": "rel/d17.1",
       "vsMajorVersion": 17,
-      "insertionTitlePrefix": "[d17.1p2]"
+      "insertionCreateDraftPR": true,
```

Comment:
Mark the PR as a draft since we snap early than VS, and rel/d17.1 is not open yet